.circleci/config.yml

```diff
@@ -542,6 +547,65 @@ jobs:
       <<: *UTF_8_ENVIRONMENT
       TAHOE_LAFS_TOX_ENVIRONMENT: "py310"
 
+  debian-13-package:
+    docker:
+      - <<: *DOCKERHUB_AUTH
+        image: "debian:trixie"
+    steps:
+      - run:
+          name: "Get latest Debian repository index and install essentials"
+          command: |
+            apt update
+            apt-get install --no-install-recommends --yes \
+              git \
+              ca-certificates \
+              quilt
+      - run:
+          name: "Download sources for Tahoe-LAFS and @merkys Debian packaging"
+          command: |
+            mkdir /tmp/debian-13-package
+            cd /tmp/debian-13-package
+            # Take Tahoe-LAFS source
+            git clone https://github.com/tahoe-lafs/tahoe-lafs.git
+            # Take debian/ directory from Debian packaging repository
+            git clone https://salsa.debian.org/merkys/tahoe-lafs.git merkys-tahoe-lafs
+            cd tahoe-lafs/
+            ln -s ../merkys-tahoe-lafs/debian
+            ln -s ../merkys-tahoe-lafs/debian/patches
+      - run:
+          name: "Install Tahoe-LAFS dependencies from the Debian repositories"
+          command: |
+            # 'apt-get install' dependencies from debian/control
+            apt-get install --no-install-recommends --yes $(awk '
+              /^(Build-)?Depends:/ || /^ / && deps {
+                sub(/^[^ ]+: /, "")
+                deps = 1
+                dep_str = dep_str ", " $0
+                next
+              }
+              { deps=0 }
+              END {
+                split(dep_str, dep_array, /[,|] */)
+                for (d in dep_array) {
+                  dep = dep_array[d]
+                  gsub(/[^a-z0-9_.+-].*$/, "", dep)
+                  if (dep && !seen[dep]++) print dep
+                }
+              }' /tmp/debian-13-package/tahoe-lafs/debian/control)
+      - run:
+          name: "Apply the Debian package's patches"
+          command: |
+            cd /tmp/debian-13-package/tahoe-lafs/
+            # TODO decide: Should we honour quilt's opinion about this build?
+            quilt push -a || true
```

Comment:
Converted to draft because of this line:  Should applying the package's patches not working break the build or not?

Reply:
Since the point of this builder is to test the debian 13 package it seems to me the obvious answer is "yes the build fails if the pathches fail to apply". 

However overall this builder should be considered " optional" I think (that is: if this trixie build fails the PR / etc is still green)

Reply:
Hm.  I think this builder can be more useful if we use it to ensure our compatibility with Debian 13.
I try to change the title accordingly.

Reply:
The current builder output has warnings we probably want to take a closer look at - I'll file tickets for these.

Also I found very cool/interesting - did you see? - that this builder tests both, Python 3.12 and Python 3.13, and both succeed.